src/content/components/FeatureView/FeatureView.js

```diff
@@ -124,7 +124,9 @@ const EngineeringView = props => {
                 key={meta.id}
                 subtitle={removeMeta(meta.summary)}
                 meta={meta.id}
-                fileNew={`blocked=${meta.id}, ${parentMeta}`}
+                fileNew={`blocked=${meta.id},${parentMeta}&component=${
+                  parentMeta.component
```

Comment:
can you change this to `meta.component` since `parentMeta` is apparently a bug number